src/bundle/Resources/config/services/controllers.yaml

```diff
@@ -216,5 +216,10 @@ services:
     Ibexa\Bundle\AdminUi\Controller\Permission\LanguageLimitationController:
         parent: Ibexa\Contracts\AdminUi\Controller\Controller
         autowire: true
+
+
+    Ibexa\Bundle\AdminUi\Controller\ApplicationConfigController:
```

Comment:
Do we have already another REST controllers in this namespace? Asking because I'd prefer some separation, like `Ibexa\Bundle\AdminUi\Controller\REST\ApplicationConfigController`, but if that's pre-existing pattern here, then that's fine I guess.

Reply:
Yes, unfortunately we have already added REST controllers in this namespace. For consistency I did not added `ApplicationConfigController` in new namespace.